src/data/markdown/docs/01 guides/05 Testing Guides/04 Running large tests.md

```diff
@@ -132,7 +131,7 @@ Here's a screenshot of 3 terminal sessions showing k6, iftop and htop.
 ![k6 iftop htop](./images/large-scale-testing-3-terminals.png)
 
 
-## Additional k6 flags to achieve better performance
+## k6 Options
 
```

Comment:
If you want to change the header, then I think we should add this sentence below:

`The k6 settings listed below will unlock additional performance benefits when running large tests.`

Reply:
https://github.com/loadimpact/k6-docs/pull/25/commits/6f03166368b5b118f89fc4805d93c533c98d48a4